src/subgraph/orders/index.ts

```diff
@@ -0,0 +1,21 @@
+import { request } from "graphql-request"
+import { Chain } from "../../common/chain";
+import { getSubgraphEndpoint } from "../common"
+import { fetchOrdersByUserQuery, fetchPendingOrdersQuery } from "./subgraphQueries"
+import { mapOrdersArrayToInterface } from "../common/mapper"
+
+// Get all order by a user address 
+export const getAllOrdersByUserAddress = async (chainId: Chain, userAddress: string, count: number = 10) => {
+    const subgraphEndpoint = getSubgraphEndpoint(chainId)
+
+    const subgraphResponse = await request(subgraphEndpoint, fetchOrdersByUserQuery(userAddress, count));
+    return mapOrdersArrayToInterface(subgraphResponse);
+}
+
+// Get all pending orders that are not yet settled/cancelled or expired 
+export const getAllPendingOrders = async (chainId: Chain, timestamp: number = Date.now() / 1000, count: number = 10) => {
```

Comment:
Default value for timestamp is wrong. Need to add Math.floor()

Reply:
Fixed